URA-695/venn_diagram_plots.py

```
@@ -0,0 +1,42 @@
+#reported in each output from bcftools isec
+#single clinical sample mutect2 (bam to vcf)
+import matplotlib.pyplot as plt
+from matplotlib_venn import venn2
+import pandas as pd
+import io
+import os
+#function to read vcfs that were output from bcftools isec
+# Things to do to polish the code
+# make it save a file for each sample and it its appropriate directory
+
+def read_vcf(path):
```

Ruff (0.8.0)

Missing return type annotation for public function `read_vcf`

(ANN201)